### PR TITLE
feat: DropDown menu for Availability status (AR-2674)

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2674" title="AR-2674" target="_blank">StoryAR-2674</a>  Drop down Menu for status
  </td></table>
  <br />
 

# What's new in this PR?

In SelfUserProfileScreen changing availability status should be implemented by DropDown menu, instead of separate buttons

### Attachments (Optional)

https://user-images.githubusercontent.com/6539347/201137901-134c783d-918a-448b-a2e1-c5eb945d857d.mp4

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1151/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4912789)(4912789)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4912789)(4912789)-dev-release.apk)

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1151/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4944980)(4944980)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4944980)(4944980)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt [hunk 1]
```diff
@@ -256,82 +253,35 @@ private fun CurrentSelfUserStatus(
...
-        modifier = modifier,
+    val items = listOf(
+        UserAvailabilityStatus.AVAILABLE,
+        UserAvailabilityStatus.BUSY,
+        UserAvailabilityStatus.AWAY,
+        UserAvailabilityStatus.NONE
     )
+
+    Column(modifier = Modifier.fillMaxWidth()) {
+        FolderHeader("Availability")
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt]: We should localize this.

**Review by gongracr — APPROVED** [review]
Looking good. Just a small comment for string localisation.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1151/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4945602)(4945602)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4945602)(4945602)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt (hunk 1 above)
Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt]: right!

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1151/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1151/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4946477)(4946477)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4946477)(4946477)-dev-release.apk)

**Pull request merged**